test/relay/relay.node.ts

```diff
@@ -157,32 +159,30 @@ describe('Dialing (via relay, TCP)', () => {
 
     // send an invalid relay message from the relay to the destination peer
     const connections = relayLibp2p.getConnections(dstLibp2p.peerId)
-    const stream = await connections[0].newStream(RELAY_CODEC)
-    const streamHandler = new StreamHandler({ stream })
-    streamHandler.write({
-      type: CircuitRelay.Type.STATUS
-    })
-    const res = await streamHandler.read()
-    expect(res?.code).to.equal(CircuitRelay.Status.MALFORMED_MESSAGE)
-    streamHandler.close()
+    // this should fail as the destination peer has HOP disabled
+    await expect(connections[0].newStream(RELAY_V2_HOP_CODEC))
+      .to.be.rejected()
```

Comment:
Please add an assertion to ensure the correct error occurs:


```suggestion
    await expect(connections[0].newStream(RELAY_V2_HOP_CODEC)).to.be.rejected()
      .with.property('code').that.equals('ERR_SOME_ERROR_CODE')
```